_episodes/15-transferring-files.md

```diff
@@ -167,6 +167,14 @@ to transfer files.
 > files.
 {: .callout}
 
+> ## A note on rsync
```

Comment:
I suggest to put this call-out at [the end](https://github.com/hpc-carpentry/hpc-intro/blob/a41c3558f3e0716630b1f29993863c3dcf2faa1a/_episodes/15-transferring-files.md#transferring-single-files-and-folders-with-scp) of the `scp` section.

Reply:
any opinions?